packages/relay/src/lib/eth.ts

```diff
@@ -1127,6 +1127,10 @@ export class EthImpl implements Eth {
     return tag == null || tag === EthImpl.blockLatest || tag === EthImpl.blockPending;
   }
 
+  private static toHex32Byte(input: string): string {
+    return input.length === 66 ? input : EthImpl.emptyHex + this.prune0x(input).padStart(64, '0');
```

Comment:
Instead of hardcoding `64` here I would go with `64 - this.prune0x(input).length`

Reply:
Yeah you're right, sorry, I misunderstood `padStart`